airbyte-integrations/connectors/source-google-ad-manager/source_google_ad_manager/utils.py

```diff
@@ -0,0 +1,6 @@
+
+def convert_time_to_dict(date):
+    return {
+        'year': date.year,
+        'month': date.month,
+        'day': date.day}
```

Comment:
do you need a specific file to only one function? move this to where you need to use.